tests/unit/test_bigquery_adapter.py

```diff
@@ -996,18 +996,17 @@ def test_grant_access_to_does_not_call_update_with_duplicate_access_entry(self):
         self.adapter.grant_access_to(
             entity=self.entity, entity_type="table", role=None, grant_target_dict=grant_target_dict
         )
-        print(self.mock_connection.call_args)
-        print(self.mock_client.call_args)
         self.mock_client.update_dataset.assert_not_called()
 
     def test_grant_access_to_calls_update_with_valid_access_entry(self):
         a_different_entity = BigQueryRelation.from_dict(
-        {
-            "type": None,
-            "path": {"database": "another-test-project", "schema": "test_schema_2", "identifier": "my_view"},
-            "quote_policy": {"identifier": True},
-        }
-    )
+            {
+                "type": None,
+                "path": {"database": "another-test-project",
```

Comment:
This is minor. I would either put path on one line (but not if it's longer than 80 char):
```"path": {"database": "another-test-project", "schema": "test_schema_2", "identifier": "my_view"},```
or put an entry per line:
```
"path": {
    "database": "another-test-project",
    "schema": "test_schema_2",
    "identifier": "my_view"
}
```